src/config.rs

```diff
@@ -159,19 +159,19 @@ pub struct Config {
 
 impl Config {
     /// Load the config from a `&str`
-    pub fn from_str(raw: &str) -> Result<Config, ConfigError> {
+    pub fn load_from_str(raw: &str) -> Result<Config, ConfigError> {
```

Comment:
@thewizzy thoughts on implementing from_str on it via [the trait](https://doc.rust-lang.org/std/str/trait.FromStr.html)? Do you think that'd be a lot of extra work?

Reply:
I don't think it would be considerable work. This one did give me pause for a sec, I actually started implementing `from_str` trait but it didn't feel right. Happy to take your preference.
It is really just wrapping that function in `impl FromStr for Config `

Reply:
There is a new commit moving to the `from_str` trait. Take your pick.